buffer/src/test/java/io/netty/buffer/PooledByteBufAllocatorTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.buffer;
+
+import io.netty.util.internal.SystemPropertyUtil;
+import org.junit.Assume;
+import org.junit.Test;
+
+import java.util.ArrayDeque;
+import java.util.Queue;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
+
+public class PooledByteBufAllocatorTest {
+
+    @Test
+    public void testConcurrentUsage() throws Throwable {
+        // Only run this test if explicit asked to do so as it will run for a minute.
```

Comment:
Actually the plan was to always run it on the ci. We could also reduce the time to 30 secs and run it always. As this is a race I wanted to have it run a bit, so I choose 1min. That said most of the tomes it failed within 15 secs. WDYT?

> Am 26.10.2015 um 03:09 schrieb Scott Mitchell <notifications@github.com>:
> 
> In buffer/src/test/java/io/netty/buffer/PooledByteBufAllocatorTest.java:
> 
> > +import io.netty.util.internal.SystemPropertyUtil;
> > +import org.junit.Assume;
> > +import org.junit.Test;
> > +
> > +import java.util.ArrayDeque;
> > +import java.util.Queue;
> > +import java.util.concurrent.ExecutorService;
> > +import java.util.concurrent.Executors;
> > +import java.util.concurrent.TimeUnit;
> > +import java.util.concurrent.atomic.AtomicReference;
> > +
> > +public class PooledByteBufAllocatorTest {
> > +
> > +    @Test
> > +    public void testConcurrentUsage() throws Throwable {
> > +        // Only run this test if explicit asked to do so as it will run for a minute.
> Is there anyway that we can have it run all the time and reduce the amount of time required for it to run?
> 
> —
> Reply to this email directly or view it on GitHub.
>